internetarchive/openlibrary
# Fixes Publishers page displaying only 2 pages of works

### Description
Fixes Publishers page displaying only 2 pages of works

Closes #2287 

### Technical
Changed the default pagesize of covers rendered per page of a carousel for subjects to 6.

### Testing
1. Login as admin.
2. Add 16 works by the same publisher.
3. View the publisher's page via `/publishers/<Publisher>`.
4. Make sure that the count of works matches the number of covers present in the carousel.
5. View the publisher's work in search via `/search?q=publisher%3A"<Publisher>"&mode=everything`.
6. Make sure that the count of works matches the number of works rendered.

### Evidence
Page 1:
![6coversPage1](https://user-images.githubusercontent.com/7445503/66249321-70800d80-e6ff-11e9-8b4a-7f5e5433d96e.png)

Page 2:
![6coversPage2](https://user-images.githubusercontent.com/7445503/66249324-7970df00-e6ff-11e9-80af-80912700fe76.png)

Page 3:
![6coversPage3](https://user-images.githubusercontent.com/7445503/66249326-7e359300-e6ff-11e9-859d-89c54dbc4316.png)

Search Page:
![search](https://user-images.githubusercontent.com/7445503/66249327-82fa4700-e6ff-11e9-8b33-b9fce5a736c1.png)
